app/PharBuilder.php

```diff
@@ -455,7 +455,7 @@ protected function addDir($directory, $excludes = array())
              *
              * @var \Symfony\Component\Finder\SplFileInfo $file
              */
-            $this->addFile($directory . DIRECTORY_SEPARATOR . $file->getRelativePathname());
+            $this->addFile(rtrim($directory, '\/') . DIRECTORY_SEPARATOR . $file->getRelativePathname());
```

Comment:
IMO `DIRECTORY_SEPARATOR` makes the paths unreadable. It's unnecessary clutter. I prefer always `/`, dunno whether it normalizes `\\`, too, as it's a valid path element on *nix.